File: spring-delivery-mgmt/src/main/java/app/customer/natural_customer/NaturalCustomer.java
package app.customer.natural_customer;

import app.customer.Customer;

import javax.persistence.DiscriminatorValue;
import javax.persistence.Entity;

/**
 * Created by Rafael Leal on 12/10/2018.
 */
@Entity
@DiscriminatorValue("NATURAL")
public class NaturalCustomer extends Customer {

    private String cpf;

    public String getCpf() {
        return cpf;
    }

    public void setCpf(String cpf) {
        this.cpf = cpf;
    }
}

File: cwbmess-delivery-mgmt-panel/src/app/domain/customer/natural-customer.ts
import { Customer } from './customer';
import { CustomerTypeEnum } from '../enums';
import { Address } from '../address';

export class NaturalCustomer extends Customer {
    cpf: string;

    constructor() {
        super();
        this.address = new Address();
        this.customerType = 'NATURAL';
    }
}
